pkg/polaris/ctl_polaris.go

```diff
@@ -206,6 +207,22 @@ func (ctl *CRSpecBuilderFromCobraFlags) SetCRSpecFieldByFlag(f *pflag.Flag) {
 			ctl.spec.PolarisDBSpec.PostgresDetails.Password = ctl.PostgresPassword
 		case "postgres-size":
 			ctl.spec.PolarisDBSpec.PostgresDetails.Storage.StorageSize = ctl.PostgresSize
+		case "postgres-container":
+			ctl.spec.PolarisDBSpec.PostgresDetails.IsInternal = ctl.PostgresInternal
+			ctl.spec.PolarisDBSpec.PostgresDetails.SSLMode = PostgresSSLModeDisable
+		case "postgres-ssl-mode":
+			switch PostgresSSLMode(ctl.PostgresSSLMode) {
+			case PostgresSSLModeDisable:
+				ctl.spec.PolarisDBSpec.PostgresDetails.SSLMode = PostgresSSLModeDisable
+			//case PostgresSSLModeAllow:
```

Comment:
If these cases aren't supported, we shouldn't list them in the help message for the flag

Reply:
Good catch. Fixed